jupyter_core/tests/test_paths.py

```diff
@@ -273,6 +274,39 @@ def test_is_hidden_win32():
         assert is_hidden(subdir1, root)
         assert is_file_hidden(subdir1)
 
+@pytest.mark.skipif(
+    not (sys.platform == "win32" and "__pypy__" in sys.modules),
+    reason="only run on windows/pypy: https://foss.heptapod.net/pypy/pypy/-/issues/3469"
```

Comment:
since we're using this twice, should we hoist it to a reusable mark?

Reply:
oh, it's _not_ the same w/r/t `not` and `==`! is fine

Reply:
maybe add a `and sys.implementation.version < (7, 3, 6)` since after the PyPy 7.3.6 release this should be fixed (nit: there was a bug in `sys.implementation.version` where PyPy used the Python version not the implementation version, but that is all behind us now, and the less-than comparison will still work properly even with the bug)

Reply:
> maybe add a `and sys.implementation.version < (7, 3, 6)`

Done! Thanks!